lua/telescope/builtin/files.lua

```diff
@@ -247,7 +247,7 @@ files.file_browser = function(opts)
         table.insert(data, typ == 'directory' and (entry .. os_sep) or entry)
       end
     })
-    table.insert(data, 1, '../')
+    table.insert(data, 1, '..\\')
```

Comment:
this should be `'..' .. os_sep` so it works for both Linux and Windows

Reply:
I changed it to your suggested solution. (Thx for the hint.)